src/main/java/org/threeten/extra/Temporals.java

```diff
@@ -474,4 +467,14 @@ public static Duration multiply(Duration duration, double multiplicand) {
         return durationFromBigDecimalSeconds(amount);
     }
 
+    /**
+     * Useful Duration constants expressed as BigDecimal seconds with a scale of 9.
+     */
+    private static final class BigDecimalSeconds {
+        public static final BigDecimal MIN = BigDecimal.valueOf(Long.MIN_VALUE).add(BigDecimal.valueOf(000_000_000, 9));
```

Comment:
Intellij warning: `000_000_000` is an octal integer. Convert to decimal `0`.

The value is zero either way, but I thought I'd mention it just in case.

Reply:
Thanks, fixed! The less warnings, the better.